decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Elections
+    # The data store for a document in the Decidim::Elections component. It stores a
+    # title, description and any other useful information to render a custom
+    # document.
+    class Election < Elections::ApplicationRecord
+      include Decidim::Resourceable
+      include Decidim::SoftDeletable
+      include Decidim::HasComponent
+      include Decidim::HasAttachments
+      include Decidim::Publicable
+      include Decidim::Traceable
+      include Decidim::TranslatableResource
+      include Decidim::TranslatableAttributes
+      include Decidim::Loggable
+      include Decidim::Searchable
+      include Decidim::Reportable
+
+      component_manifest_name "elections"
+
+      translatable_fields :title, :description
+
+      validates :title, presence: true
+
+      scope :published, -> { where(published_at: ..Time.current) }
```

Comment:
I am not sure why you are using this approach, but we should not use it. Use #13289 & #13291 for additional details on what this approach could generate.